invite/templates/invite/invite.html

```diff
@@ -122,14 +129,58 @@
 
         // Register the keyup event handler so the search boxes work
         $("#perms-search, #groups-search").keyup(searchOptions);
+
+        // With a new form, make sure only the first form row has the "add" button
+        $("#invitees button#add:gt(0)").remove();
+
+        // Check to see if the username or email address is already in use, and display error if so
+        function validateField() {
+            var xhttp = new XMLHttpRequest(),
```

Comment:
Style nitpick. You can do it one way or another, as long as it's consistent, but I prefer declaring each variable on its own line with the declaration keyword. I think it makes it easier to recognize the variable declarations and definitely makes it easier if you have to add or delete variables from the code block, you don't find yourself messing between commas and semi-colons at the end of the line, it's always a semi-colon.

```
var xhttp = new XMLHttpRequest();
var $input = $(this);
var $row = $input.closest(".control-group");
var type = $input.attr("placeholder").toLowerCase();
var usernameError = $("<li></li>").text("Username: Username taken, choose another");
var emailError = $("<li></li>").text("Email: The email provided already belongs to a user");
var typeToError = {"email":emailError, "username":usernameError};
var error = typeToError[type];
```

Reply:
gotcha, I'll give them each their own full declaration